modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -118,6 +118,10 @@ public void create(
 			Map<String, Serializable> parameters)
 		throws Exception {
 
+		for (ObjectDefinition objectDefinition : objectDefinitions) {
+			objectDefinition.setActive(false);
```

Comment:
Why do you add a new **for** statement to perform it, instead of do it inside the existing **for** placed after `super.create(...` line?